internal/pkg/kube/gateway_reconciler.go

```diff
@@ -46,7 +46,7 @@ func (r *GatewayResourceReconciler) GetTypeUrl() string {
 }
 
 func (r *GatewayResourceReconciler) Reconcile(ctx context.Context) error {
-	gw, err := r.cf.GetIngressGateway()
+	gw, err := r.cf.IngressGateway()
```

Comment:
nit: we have `GetWorkloadEntries` left behind with the getter naming convention

Reply:
Done